UnityProject/Assets/Scripts/Shuttles/NetworkedMatrixMove.cs

```diff
@@ -304,23 +290,26 @@ public void Awake()
 		ObjectLayer = TargetTransform.GetComponentInChildren<ObjectLayer>();
 
 
-
 		UpdateManager.Add(CallbackType.EARLY_UPDATE, UpdateLoop);
 		ElapsedTimeSinceLastUpdate.Reset();
 		ElapsedTimeSinceLastUpdate.Start();
-		OnRotate.Invoke();
+		OnRotate?.Invoke();
 		var FacedDirection = ForwardsDirection.ToOrientationEnum();
 		if (PreviousDirectionFacing != FacedDirection)
 		{
 			PreviousDirectionFacing = FacedDirection;
-			OnRotate90.Invoke();
+			OnRotate90?.Invoke();
 		}
-		SetGizmoPosition(currentLocalPivot);
 
+		SetGizmoPosition(currentLocalPivot);
 	}
 
 	public void OnDestroy()
 	{
+		OnRotate = null;
+		OnRotate90 = null;
+		OnStartMovement = null;
+		OnStopMovement = null;
```

Comment:
`RemoveAllListners()` exists on unity events

Reply:
It is no longer unity events is c# events